sites/all/modules/access/acc_applications/acc_applications.info

```diff
@@ -32,6 +32,7 @@ dependencies[] = taxonomy
 dependencies[] = text
 dependencies[] = views
 dependencies[] = views_data_export
+dependencies[] = views_pivot
```

Comment:
This won't enable the views_pivot module if it's disabled, we need a hook_update_N to do so.